larray/tests/test_array.py

```diff
@@ -2652,6 +2652,16 @@ def test_binary_ops(small_array):
     assert raw2_ge_la.axes == small_array.axes
     assert_array_equal(raw2_ge_la, raw2 >= raw)
 
+    # arrays filled with None
+    arr = full(small_array.axes, fill_value=None)
+    res = arr == None                                                       # noqa: E711
+    assert_array_equal(res, full(small_array.axes, fill_value=True))
```

Comment:
use ones(small_array.axes) instead of full(..., fill_value=True)?

Reply:
OK but why ?

Reply:
Because it is a bit shorter and faster and if even we don't use those specialized functions, we could as well remove them.
FWIW, to be exactly equivalent, it should be 
```python
ones(small_array.axes, dtype=bool)
```